samples/eshop/eshop.bicep

```diff
@@ -0,0 +1,220 @@
+import radius as rad
+
+// Parameters -------------------------------------------------------
+
+@description('Radius environment ID. Set automatically by Radius')
+param environment string
+
+@description('Container registry to pull from, with optional path. Defaults to "ghcr.io/radius-project/samples/eshop"')
+param imageRegistry string = 'ghcr.io/radius-project/samples/eshop'
+
+@description('Container image tag to use for eshop images. Defaults to "latest".')
+param imageTag string = 'latest'
```

Comment:
@willdavsmith had to update this back to latest since we didn't have a linux-dotnet7 tag in our package list